WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -3,31 +3,37 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="title" content="Wasabi Wallet - Unfairly Private">
+    <meta name="title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta name="description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
 
     <!-- Open Graph / Facebook -->
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://www.wasabiwallet.io">
-    <meta property="og:title" content="Wasabi Wallet - Unfairly Private">
+    <meta property="og:title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta property="og:description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
     <meta property="og:image" content="https://www.wasabiwallet.io/images/wasabi-social.jpg">
 
     <!-- Twitter -->
     <meta property="twitter:card" content="summary_large_image">
     <meta property="twitter:url" content="https://www.wasabiwallet.io">
-    <meta property="twitter:title" content="Wasabi Wallet - Unfairly Private">
+    <meta property="twitter:title" content="Wasabi Wallet - Unfairly Private Bitcoin Wallet">
     <meta property="twitter:description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
     <meta property="twitter:image" content="https://www.wasabiwallet.io/images/wasabi-social.jpg">
 
     <link href="https://fonts.googleapis.com/css?family=Inconsolata:400,700|Playfair+Display:700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/bootstrap.css?ver=1.1.0">
     <link rel="stylesheet" href="css/style.css?ver=1.1.0">
     <link rel="shortcut icon" href="images/favicon.ico" />
-    <title>Wasabi Wallet - Unfairly Private</title>
+    <title>Wasabi Wallet - Unfairly Private Bitcoin Wallet</title>
```

Comment:
Oh boy, these ` Bitcoin Wallet` additions is a classical SEO ruins the design case. I think we shouldn't do it.

Reply:
Ok, not really necessary but should help for SEO.
I'll remove it